pkg/storage/mvcc_incremental_iterator_test.go

```diff
@@ -115,7 +196,7 @@ func TestMVCCIncrementalIterator(t *testing.T) {
 		testValue3 = []byte("val3")
 		testValue4 = []byte("val4")
 
-		tsMin = hlc.Timestamp{WallTime: 0, Logical: 0}
+		tsMin = hlc.Timestamp{WallTime: 0, Logical: 1}
```

Comment:
I was a bit confused about why we needed to up the logical timestamp for the export tests. I think it's odd enough that it may warrant a brief explanation as a comment?

Reply:
Good point, done!